src/QsCompiler/Transformations/SearchAndReplace.cs

```diff
@@ -446,6 +446,166 @@ public ExpressionKindTransformation(SyntaxTreeTransformation<TransformationState
         }
     }
 
+    /// <summary>
+    /// A transformation that renames all references to each given qualified name.
+    /// </summary>
+    public class RenameReferences : SyntaxTreeTransformation
```

Comment:
It could be, but it fits with the rest of the transformations in this file, and having both SearchAndReplace.cs and ReferenceRenaming.cs might be confusing since they mean similar things. So I'm not sure how to reorganize the other transformations properly.